source/howto_automatic-deployment.rst

```diff
@@ -29,16 +29,25 @@ There are several solutions available to automatically deploy your code to your
 - GitLab CI/CD
 - Gitea Actions
 
-.. note:: For this guide you should be familiar with the basic concepts of git_.
+.. note:: For this guide you should be familiar with the basic concepts of git.
 
-Prepare Local Git Repository
+Prerequisites
+============
+Install or make sure the following command line tools are installed
+
+    - Git
```

Comment:
When using uberspace, yes. But I thought about a scenario when you have your code on your local machine or git repository. 

I don't think that it makes so much sense to use git from your uberspace directly. If you would do it directly on your uberspace you might don't need a git repository any way. 

When using Windows (like me) git is not installed. 

So you suggest, that we also include a detailed installation guide of git? This would also then include: 
- Installation (when on windows)
- SSH Key Setup

Describing that Git and Github CLI knowledge is required is not enough.

Reply:
Good point, my fault. 

But to make it clearer, then maybe add 'on your local machine'  to 'install or make sure the following command line tools are installed' and additionally  prefix the code snippets with a '[you@yourhost ~]$'-prompt analog to '[isabell@stardust ~]$' .

Reply:
So no complete Git Tutorial? 

Just mention: 
- Is git installed? Check with git --version 
When not installed, refer to git documentation